javascript-exercises-1/booklist/booklist.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Book List</title>
+</head>
+<body>
+  <h1>My Book List</h1>
+</body>
+<script src="./booklist.js"></script>
```

Comment:
@JennyDoJD Why `script` tag at the bottom of the file? Can we move it to the top?

Reply:
btw, help me list out attributes of `script` tag

Reply:
We could move it to the top but the example request was to put it at the bottom of the page so I left it there

Reply:
No, it's different between the top and bottom. Pls do more research

Reply:
@JennyDoJD Do you have any output here?

Reply:
Attributes of `script ` tag:
1. src: Specifies the URL of the external script file to be imported.
2. type: Specifies the MIME type of the script. Common values include "text/javascript" and "module".
3. defer: Specifies that the script should be executed after the document has been parsed.
4. async: Specifies that the script should be executed asynchronously as soon as it is available.
5. charset: Specifies the character encoding used in the external script file.

Reply:
Output: 
![image](https://github.com/JennyDoJD/javascript-training/assets/110828723/13a5707c-9863-446f-85a4-4c4b4db4d0e7)